tests/test_regression.py

```diff
@@ -45,3 +45,23 @@ def test_p4_multiple_spaces_in_mappings(perforce_workspace, tmpdir):
     environment.settings.PerforceMainVcs.force_clean = True
     environment.settings.PerforceWithMappings.mappings = [f"{perforce_workspace.depot}   /..."]
     assert not __main__.run(environment.settings)
+
+
+def test_non_utf8_environment(docker_main):
+    output = docker_main.run("""
+from universum.configuration_support import Variations
```

Comment:
From the test code it looks like POSIX locale has UTF8 encoding (codeset) and en_US doesn't have it.
This is counterintuitive. 
If the test is correct, please provide explanations in source code comments.